source/_integrations/thread.markdown

```diff
@@ -14,30 +14,158 @@ ha_platforms:
+
+### A communication protocol for IoT devices
+
+Thread is a low-power mesh networking standard for IoT devices. The low-power aspect is important for battery-powered smart home devices. However, it's also low-bandwidth, making it ideal for applications that don't send a lot of data, like switches or motion sensors.
+
+Thread uses the same <abbr title="radio frequency">RF</abbr> technology as Zigbee (IEEE 802.15.4) but provides IP connectivity similar to Wi-Fi. Unlike Zigbee, Thread by itself does not allow controlling devices: It is just a communication protocol. To control the Thread devices, a higher-level protocol is required: Matter or Apple HomeKit. Thread devices use the IPv6 standard to communicate both inside and outside the mesh network.
+
+### About Thread border routers
+
+The devices use Thread border routers to communicate outside the mesh with any IPv6-capable device. A Thread border router is connected to your network either via Wi-Fi or Ethernet and uses its <abbr title="radio frequency">RF</abbr> radio to communicate with the Thread mesh network. The <abbr title="Thread border router">TBR</abbr> routes packets between your local network and the Thread mesh. It does not look at the content of these packets, it just forwards them.
+
+Often, Thread border routing is only an auxiliary functionality of a smart home device. For example, the Nest Hub (2nd gen) is a smart display, a Google Home controller, and a Chromecast target, but also has a Thread border router included.
+
+Unlike other protocols, Thread can use multiple border routers in a single network. This increases wireless coverage and reduces the risk of a single point of failure. Ideal for home automation, with a potentially large number of devices spread over a large area.
+
+OpenThread is an open source implementation of Thread, originally released by Google. Almost all commercially available Thread border routers are based on the open source implementation. However, the configuration of <abbr title="Thread border routers">TBRs</abbr> is not part of the Thread standard. This means that Google and Apple <abbr title="Thread border routers">TBRs</abbr> implementation setup and configured by their respective ecosystems. 
+
+Home Assistant can only control OpenThread border routers built with the REST API available in the open source implementation. The OpenThread Border Router add-on (as well as the OpenThread Border Router bundled in the Silicon Labs Multiprotocol add-on) are built from this open source OpenThread code and have the REST API enabled.
+
+### List of Thread border router devices
+
+Currently, the following <abbr title="Thread border router">TBR</abbr> devices are known to work well with Home Assistant:
+
+#### Nabu Casa
+
+- [Home Assistant Yellow](/yellow/) hub
+- [Home Assistant SkyConnect](/skyconnect/) USB stick. Out of the box, it runs Zigbee, but you can make it [run Thread](https://skyconnect.home-assistant.io/procedures/enable-thread/) instead.
```

Comment:
we need to move this into another place. These are not known to work well but are still in development.

Reply:
Simple solution would be to move them under Apple and Google and add a notice that using these is still in development and not all paths work as worry-free as using one of the commercially available border routers. We really need to set the right expectations.

Reply:
From a Thread perspective, these are working perfectly fine. I (and countless others) have them running just fine.

The problem is the integration with Matter, specifically with the iOS eco-system. And I agree we should add a warning/note here.

I'd also leave them at the top, just like our hubs in the getting started section.

Reply:
I don't agree because of the first sentence "these are known to work well with Home assistant". The only ones known to work well (as in: stable and all paths covered) are existing border routers from both apple and google.

Using a SkyConnect as PTBR may work well for you and some others but that does not make it a recommended solution. It is all about setting expectation and giving especially newcomers the good advise. People that want to run their own thread networks using their own custom flashed OTBR's are probably the ones that dont need these docs.

Reply:
> "these are known to work well with Home assistant"

What do you mean? It works super really well here.

>  (as in: stable and all paths covered) are existing border routers from both apple and google.

Well, my first BR I got a bit more than a year ago was an Apple 4K which I got from my TV provider. I am in Android land. I tried to make it work but there is just no way to add any device if you don't own an iOS phone. :see_no_evil:  Not sure how it is the other way around (can you setup a Google TBR with iOS? :thinking: )

In the end, especially for Matter, all of those listed TBR need a particular combination of things to be able to work. I would really prefer to document and make recommendation in the protocol specific part, because it is protocol dependent what works best.

Reply:
Fair enough:

**Apple Border Routers:**
Recommended only for people that are already on the Apple ecosystem, having at least a recent iPhone but maybe also other Apple gear. 

**Google Border Routers:**
Recommended only for people that are on the Android ecosystem, having at least a recent Android phone and an existing Nest Hub (v2) OR Google Nest WiFi pro. If no existing Google border router present (and also no desire to add one later), use OTBR (see below).

** Home Assistant Border Router**
The Home Assistant SkyConnect/Yellow with the OTBR add-on can be used as a completely independent solution but is currently limited to users on the Android ecosystem and one single BR. Support for multiple BR's will be added soon.


Something along those lines makes it a fair comparison

Reply:
The intention of this section is to provide a simple list of devices that can act as a border router.
It is not a buying recommendation or instruction on how to set up a border router or Thread network.

Because the prereqs (which Companion app or phone you need, etc) depend on the protocol, I suggest to add a link to these sections, rather than overloading the list with all this information.

Reply:
Let's keep the list on the thread docs flat and easy and move the matter related recommendations to the matter docs (and we can now also remove some duplication there that is now covered very well in this thread docs)